tests/VCR/VCRCleanerEventSubscriberTest.php

```diff
@@ -183,4 +183,51 @@ public function testCurlCallWithRedactedHostname()
         $this->assertNotContains('www.example.com', $vcrFile);
         $this->assertContains('[redacted]', $vcrFile);
     }
+
+    public function testCurlCallToModifyResponseHeaders()
+    {
+        $newFile = $this->getCassetteContent();
+        $this->assertEmpty($newFile);
```

Comment:
This is getting repetitive in each test. Is this even necessary? If so, can I move it to `setup()` instead?